overlord/snapstate/autorefresh.go

```diff
@@ -536,30 +549,53 @@ func (m *autoRefresh) launchAutoRefresh() error {
 		return nil
 	}
 
```

Comment:
do I understand correctly that this now can create two changes? a pre-download one and a autorefresh one?